src/lucky_web/tags/select_helpers.cr

```diff
@@ -0,0 +1,24 @@
+module LuckyWeb::SelectHelpers
+  alias SelectOption = (String | Int32 | Int64)
+
+  def select(field : LuckyRecord::AllowedField, **html_options)
+    select_tag merge_options(html_options, {"name" => input_name(field)}) do
+      yield
+    end
+  end
+
+  def options_for_select(field : LuckyRecord::AllowedField, select_options : Array(String, SelectOption), **html_options)
+    select_options.each do |select_name, select_value|
+       is_selected = if select_value.to_s == field.value.to_s
```

Comment:
Sorry for the barrage of comments, but I just remembered that you should use `field.param` here instead. That's because if a field fails to save, the `param` method will be set, but value will be `nil`.